src/main/webapp/app/shared/components/button.component.ts

```diff
@@ -52,5 +53,7 @@ export class ButtonComponent {
     @Input() isLoading = false;
     @Input() featureToggle: FeatureToggle; // Disable by feature toggle.
 
+    @Input() shouldSubmit = true;
```

Comment:
Can you do it as enum of types, like we have for `buttonType` already? Would be nice to stay consistent and there can also be the case that we need reset button type.